modules/ding_ting_frontend/ding_ting_frontend.pages_default.inc

```diff
@@ -564,36 +564,6 @@ search/ting
     $display->content['new-b00e0b46-6043-48c2-98ad-6446c489e7f9'] = $pane;
     $display->panels['left_sidebar'][0] = 'new-b00e0b46-6043-48c2-98ad-6446c489e7f9';
     $pane = new stdClass();
-    $pane->pid = 'new-26a27e6c-f934-4169-a06e-04db0f61ebf6';
-    $pane->panel = 'left_sidebar';
-    $pane->type = 'campaign';
-    $pane->subtype = 'campaign';
```

Comment:
This removed a campaign pane. Is that relevant to the change at hand?

Reply:
As I said before, no, but cherrypicking in features just passes the buck.